R/neha.R

```diff
@@ -7,6 +7,15 @@
 #' @return A data frame in which, in addition to all of the variables in 'eha_data', there is one column for each directed dyad, named 'i_j', where 'i' and 'j' are node ids, in which the value indicates the number of time points before 'time' that 'i' experienced the event in the respective 'cascade'. If 'i' did not experience the cascade before 'time', the value is 0.
 #' @export
 data_neha_discrete <- function(eha_data, node, time, event, cascade){
+  
+  # Ensure data is in correct format
+  if (!inherits(eha_data, "data.frame")) {
```

Comment:
Nit: My R is a bit rusty tbh, but I think this check is unnecessary because you can call `as.data.frame` on a data frame and it will just return the dataframe